Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:8.15.0
+
+# Install VS Code's deps. These are the only two it seems we need.
+RUN apt-get update
+RUN apt-get install -y libxkbfile-dev libsecret-1-dev
+
+# Ensure latest yarn.
+RUN npm install -g yarn
+
+# In the future, we can use https://github.com/yarnpkg/rfcs/pull/53 to make it use the node_modules
+# directly which should be faster.
+WORKDIR /src
+COPY . .
+RUN yarn
+RUN yarn task build:server:binary
+
+# We deploy with ubuntu so that devs have a familiar environemnt.
+FROM ubuntu
+RUN apt-get update
```

Comment:
Compress these two `RUN`s as one instruction.

Reply:
Its a small thing but I like em separate to make better use of docker's layer cache.

Reply:
Optimizing layers no matter how small should be done.

Reply:
@nhooyr it is fine to separate them while you are getting the image done, but once it is there, it is [strongly recommended (see “apt-get” section)](
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) to combine update + install + remove caches

Reply:
That is interesting. Does seem reasonable to me to bust the cache every time the package list is modified.